vsphere/datadog_checks/vsphere/constants.py

```diff
@@ -195,3 +197,23 @@
     'VmReconfiguredEvent': [],
     'VmSuspendedEvent': [],
 }
+
+EVENTS_FILTERABLE_BY_RESOURCE = [
```

Comment:
Can we update the docs to reflect these are filtered by event_resource_filters now? https://github.com/DataDog/integrations-core/pull/17917/files#diff-58889a87211c74e43622d0d2061da129d7566d465c368a4a89dbc2702aff00c0R250

Reply:
Updated spec.yaml.